lib/streamlit/dataframe_util.py

```diff
@@ -623,6 +690,19 @@ def convert_anything_to_pandas_df(
     if has_callable_attr(data, "to_pandas"):
         return pd.DataFrame(data.to_pandas())
 
+    if has_callable_attr(data, "toPandas"):
```

Comment:
nit: could you add a comment when this `toPandas` exists vs. `to_pandas`? It might help someone who is not familiar with it

Reply:
Decided to remove it agaon. Pyspark and Snowpark uses the camel cases naming for whatever reason. But we support both of them via optimized logic anyways and I couldn't find a lot of other relevant libraries that also use this camelCase notation.